### PR TITLE
Small Change for Sprint0

### LINKED ISSUE #3: Sean Test Issue
Test issue for Sprint0

### PR DESCRIPTION
**Pull request by ghost**
Made small change
Resolves #3

### CODE REVIEW

File: HotKey/HotKey_MainFolder/Form1.cs [hunk 1]
```diff
@@ -14,6 +14,7 @@ public partial class Form1 : Form
     {
         public Form1()
         {
+            Console.WriteLine();
```
Comment (pmdwight) [inline HotKey/HotKey_MainFolder/Form1.cs]: I don't think we need this, Sean.

**Review by pmdwight — CHANGES_REQUESTED** [review]
See comments and change.

File: HotKey/HotKey_MainFolder/Form1.cs (hunk 1 above)
Comment (ghost) [reply HotKey/HotKey_MainFolder/Form1.cs]: I deleted it please pass it. I beg.

**Review by pmdwight — APPROVED** [review]
Impeccable code, Sean.

**Comment by pmdwight** [comment]
Unfortunately you are trying to merge into master, and I simply cannot allow that. Please merge into develop.

**Pull request merged**